### PR TITLE
PMM-8308: add process exec path to AgentInfo and StateChangedRequest

### PR DESCRIPTION
**Pull request by qwest812**
Related tickets: 
 - https://jira.percona.com/browse/PMM-8308

FB: https://github.com/Percona-Lab/pmm-submodules/pull/2307

Related changes:
 - pmm-admin https://github.com/percona/pmm-admin/pull/176
 - pmm-agent  https://github.com/percona/pmm-agent/pull/314
 - pmm-managed https://github.com/percona/pmm-managed/pull/988

### CODE REVIEW

**Review by BupycHuk — COMMENTED** [review]
Please add processExecPath into inventory API.

File: api/inventorypb/agents.proto [hunk 1]
```diff
@@ -58,6 +60,8 @@ message VMAgent {
 
   // Actual Agent status.
   AgentStatus status = 5;
+  //Path to exec process
```
Comment (BupycHuk) [inline api/inventorypb/agents.proto:63]:
```suggestion
  // Path to exec process
```

File: api/inventorypb/agents.proto [hunk 2]
```diff
@@ -82,6 +86,8 @@ message NodeExporter {
   AgentStatus status = 5;
   // Listen port for scraping metrics.
   uint32 listen_port = 6;
+  //Path to exec process
```
Comment (BupycHuk) [inline api/inventorypb/agents.proto:89]:
```suggestion
  // Path to exec process
```

File: api/inventorypb/agents.proto [hunk 3]
```diff
@@ -126,6 +132,8 @@ message MySQLdExporter {
   uint32 listen_port = 10;
   // True if tablestats group collectors are currently disabled.
   bool tablestats_group_disabled = 12;
+  //Path to exec process
```
Comment (BupycHuk) [inline api/inventorypb/agents.proto:135]:
```suggestion
  // Path to exec process
```

File: api/inventorypb/agents.proto [hunk 4]
```diff
@@ -165,6 +173,8 @@ message MongoDBExporter {
   int32 collections_limit = 14;
   // Enable All collectors.
   bool enable_all_collectors = 15;
+  //Path to exec process
```
Comment (BupycHuk) [inline api/inventorypb/agents.proto:176]:
```suggestion
  // Path to exec process
```

File: api/inventorypb/agents.proto (hunk 3 above)
Comment (BupycHuk) [reply api/inventorypb/agents.proto:135]: please fix everywhere else

**Review by BupycHuk — APPROVED** [review]

File: api/inventorypb/agents.proto (hunk 4 above)
Comment (qwest812) [reply api/inventorypb/agents.proto]: fixed

File: api/inventorypb/agents.proto (hunk 3 above)
Comment (qwest812) [reply api/inventorypb/agents.proto]: fixed

File: api/inventorypb/agents.proto (hunk 2 above)
Comment (qwest812) [reply api/inventorypb/agents.proto]: fixed

File: api/inventorypb/agents.proto (hunk 1 above)
Comment (qwest812) [reply api/inventorypb/agents.proto]: fixed

**Review by BupycHuk — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Review by ritbl — APPROVED** [review]

**Pull request merged**